$OpenBSD: patch-src_init_c,v 1.1 2010/09/13 16:48:11 jasper Exp $

By default, on server redirects, use the original URL to get the
local file name. Fixes CVE-2010-2252.
Diff from upstream.

--- src/init.c.orig	Tue Sep 22 05:02:41 2009
+++ src/init.c	Mon Sep 13 10:13:32 2010
@@ -243,6 +243,7 @@ static const struct {
   { "timeout",          NULL,                   cmd_spec_timeout },
   { "timestamping",     &opt.timestamping,      cmd_boolean },
   { "tries",            &opt.ntry,              cmd_number_inf },
+  { "trustservernames", &opt.trustservernames,  cmd_boolean },
   { "useproxy",         &opt.use_proxy,         cmd_boolean },
   { "user",             &opt.user,              cmd_string },
   { "useragent",        NULL,                   cmd_spec_useragent },
